src/test/resources/tax-category-without-key.json

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "44e5e889-0878-4c30-941f-767cebebeb7f",
+  "version": 18,
+  "name": "Standard tax category",
+  "rates": [
```

Comment:
One rate is enough.

Reply:
created a new PR and please check this commit for the fix. https://github.com/commercetools/commercetools-project-sync/pull/213/commits/8030e3bdb0a9252deeb2e8a2b519333cc58d0861